### PR TITLE
[RDU-2020] Raleigh 2020 Initial Commit

### PR DESCRIPTION
**Pull request by normalfaults**
Created event and organizer list

### CODE REVIEW

File: data/events/2020-raleigh.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    image: "madeline-streicher.jpg"
+  - name: "John Mark Ivey"
+    employer: "Skyline ATS"
+    twitter: "JohnMarkIvey"
+    linkedin: "https://www.linkedin.com/in/johnmarkivey/"
+    website: "https://medium.com/@gkjohnmarkivey"
+    facebook: "https://www.facebook.com/jmivey"
+    image: "johnmark-ivey.jpg"
+
+organizer_email: "organizers-raleigh-2020@devopsdays.org" # Put your organizer email address here
```
Comment (kmugrage) [inline data/events/2020-raleigh.yml]: We've deprecated the use of the year in the email address. This should be raleigh@devopsdays.org.

Comment (normalfaults) [reply data/events/2020-raleigh.yml]: okay i'll update the pull request with the correct email address

File: data/events/2020-raleigh.yml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+description: "Devopsdays is coming to raleigh!" # Edit this to suit your preferences
+ga_tracking_id: "UA-145344467-1" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate:  # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
[the remark this answers is not in the document]
Comment (normalfaults) [reply data/events/2020-raleigh.yml]: Those dates haven't been finalized yet. I'll update the yml once we get confirmation.

**Comment by bridgetkromhout** [comment]
Slack and email are updated. Please add your dates to the yml file when you're ready for them to be listed.

**Pull request merged**